descarteslabs/scenes/tests/test_geocontext.py

```diff
@@ -1,6 +1,9 @@
 import pytest
 import unittest
 import multiprocessing
+import platform
+if platform.system() == 'Darwin':
```

Comment:
Move this down a little. Normally, we format our files using `black`, but that wasn't truly done for the client. Nevertheless, let's keep the same spirit, which means 2 empty lines between the imports and any other statements. So, you get
```
...
import platform
...
import warning


if platform.system() == 'Darwin':
    multiprocessing.set_start_method("fork")


class SimpleContext(...):
```